indigo/distribution.yaml

```diff
@@ -12684,6 +12684,21 @@ repositories:
       url: https://github.com/turtlebot/turtlebot_create_desktop.git
       version: indigo
     status: maintained
+  turtlebot_exploration_3d:
+    doc:
+      type: git
+      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
+      version: indigo
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
+      version: 0.0.1
```

Comment:
okay, thanks, I will add documentation for now and release later.

2016-08-25 17:32 GMT-04:00 William Woodall <notifications@github.com>:

> In indigo/distribution.yaml
> <https://github.com/ros/rosdistro/pull/12504#discussion_r76328821>:
>
> > @@ -12684,6 +12684,21 @@ repositories:
> >        url: https://github.com/turtlebot/turtlebot_create_desktop.git
> >        version: indigo
> >      status: maintained
> > +  turtlebot_exploration_3d:
> > +    doc:
> > +      type: git
> > +      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
> > +      version: indigo
> > +    release:
> > +      tags:
> > +        release: release/indigo/{package}/{version}
> > +      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
> > +      version: 0.0.1
>
> As I said, if you just want to add documentation, do not put in a release
> section. If you want to release it, so binary packages are built, you need
> to run bloom and follow the appropriate section in the "Get Involved" wiki
> page:
>
> http://wiki.ros.org/Get%20Involved#Release_Binary_Packages
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ros/rosdistro/pull/12504/files/a501576c1230317cfa52ace42d339d1a9ba47ab5#r76328821>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKFBrxFcNGVrgjkJaxA0pqa9TOkvcs-_ks5qjgnbgaJpZM4Jtg83>
> .
>